src/Include.ts

```diff
@@ -57,7 +57,7 @@ type TemplateInclude = {
 };
 
 export type PipelineRef = {
-	rules?: Pick<Rule, 'if' | 'exists' | 'changes'>;
+	rules?: Rule[];
```

Comment:
I think this is supposed to be:

```
Pick<Rule, 'if' | 'exists' | 'changes'>[]
```

I don't think the other properties of "rule" are allowed in Includes.rules: https://docs.gitlab.com/ee/ci/yaml/#includerules. I see that my mistake is not having the `[]`, so adding `[]` seems correct.